lcrs/models/decoders/actionDecoder.py

```diff
@@ -7,7 +7,7 @@
 import hydra
 import torch.nn.functional as F
 from lcrs.utils.gripper_control import world_to_tcp_frame
-from utils.distribution import Distribution
+from lcrs.utils.distribution import Distribution
```

Comment:
Have you checked if the training can run without problems after modifying the import?